libr/bin/format/dex/dex.c

```diff
@@ -36,7 +36,7 @@ RBinDexObj *r_bin_dex_new_buf(RBuffer *buf) {
 		goto fail;
 	}
 
-	r_buf_seek (bin->b, 0, 0);
+	r_buf_seek(bin->b, 0, R_BUF_SET);
```

Comment:
Imho we should reuse the io enums and add missing space

Reply:
I don't think so. If a library/app needs r_util it's not said it needs r_io as well. You should be able to use RBuffer without depending on r_io.h